sort.js

```diff
@@ -1,2 +1,12 @@
+var sortNumber = function (number) {
+  number.sort(function (a, b) {
+      if (a == b) {
```

Comment:
等価演算子（==）ではなく、厳密等価演算子を使用してください。
```suggestion
      if (a === b) {
```